tests/report/display/data/main.fmf

```diff
@@ -0,0 +1,31 @@
+/plan:
+    discover:
+        how: fmf
+    provision:
+        how: local
+    execute:
+        how: tmt
+
+/test:
+    /pass:
+        summary: Run command 'true'
+        test: 'true'
+
+# Special plan to introduce more than one guest, for verification of --display-guest.
+# Disabled by default, but enabled in the corresponding test by a magic of context.
+/multihost-plan:
+    enabled: false
+
+    discover:
+        how: fmf
+    provision:
+        - name: guest-1
+          how: container
+        - name: guest-2
+          how: container
+    execute:
+        how: tmt
+
+    adjust:
+      - when: report_multihost == yes
+        enabled: true
```

Comment:
What about just having `/plan/single` and `/plan/multi` and selecting them in `test.sh`? Seems that using the context adjust is a bit overkill here?

Reply:
Fixed in 8d37bce3.